robottelo/api/utils.py

```diff
@@ -189,3 +198,256 @@ def one_to_many_names(name):
 
     """
     return set((name, name + '_ids', Inflector().pluralize(name)))
+
+
+def configure_provisioning(org=None, loc=None):
+    """Create and configure org, loc, product, repo, cv, env. Update proxy,
+    domain, subnet, compute resource, provision templates and medium with
+    previously created entities and create a hostgroup using all mentioned
+    entities.
+
+    :param org: Default Organization that should be used in both host
+        discovering and host provisioning procedures
+    :param loc: Default Location that should be used in both host
+        discovering and host provisioning procedures
+    :return: List of created entities that can be re-used further in
+        provisioning or validation procedure (e.g. hostgroup or domain)
+    """
+    # Create new organization and location in case they were not passed
+    if org is None:
+        org = entities.Organization(name=gen_string('alpha')).create()
```

Comment:
Yeah.. that I know but intent is to create entities w/  alpha strings. Reason: if something goes wrong in test its easier to debug. However, I'll remove the ` name=gen_string('alpha')` to address the comment.